client/contracts/truffle/contracts/kns/KNSRegistry.sol

```diff
@@ -1,10 +1,12 @@
 pragma solidity 0.4.24;
 
 import "./KNS.sol";
+import "../proxy/Initializable.sol";
+
 /**
  * The KNS registry contract.
  */
-contract KNSRegistry is KNS {
+contract KNSRegistry is KNS, Initializable {
```

Comment:
If we have the V1 that supports proxy we can remove this one?

Reply:
From my understanding we need this as the next contract may also be upgraded, so it need to be Initializable. I will test if only the original contract needs to be Initializable